client/src/pages/EventsPage/EventsPage.jsx

```diff
@@ -84,12 +87,14 @@ const EventsPage = () => {
 			<div className="cards-list-container">
 				{filteredEvents.map((item) => {
```

Comment:
(don't commit this directly, i cannot suggest the full block)

(I have ordered these as they will be used in the Card)

```suggestion
				{filteredEvents.map((eventItem) => {
					const { id, title, date, category, summary } = eventItem;
					const { url, alternativeText } = eventItem.images[0];
					return (
						<Card
							key={id}
							id={id}
							image={url}
							alternativeText={alternativeText}
							title={title}
						  date={date}
              tag={category}
							summary={summary}
						/>
					);
				})}
```

Reply:
^do you want to do this?

Reply:
^do you want to do this destructuring?

(my comment was saying that I am not able to "suggest" the correct lines because GitHub says I am suggesting on deleted lines)